### PR TITLE
made gamemode kill cava and waybar and volume is at a max of 150%

### PR DESCRIPTION
**Pull request by Sipper1236**
# Pull Request

## Description

Please read these instructions and remove unnecessary text.

- Try to include a summary of the changes and which issue is fixed.
- Also include relevant motivation and context (if applicable).
- List any dependencies that are required for this change. (e.g., packages or other PRs)
- Provide a link if there is an issue related to this pull request. e.g., Fixes # (issue)
- Please add Reviewers, Assignees, Labels, Projects, and Milestones to the PR. (if applicable)

## Type of change

Please put an `x` in the boxes that apply:

- [x] **Bug fix** (non-breaking change which fixes an issue)
- [X ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected)
- [ ] **Documentation update** (non-breaking change; modified files are limited to the documentations)
- [ ] **Technical debt** (a code change that does not fix a bug or add a feature but makes something clearer for devs)
- [ X] **Other** (provide details below)
Improves performance
## Checklist

Please put an `x` in the boxes that apply:

- [ X] I have read the [CONTRIBUTING](https://github.com/JaKooLit/Hyprland-Dots/blob/main/CONTRIBUTING.md) document.
- [ ] My code follows the code style of this project.
- [ X] My commit message follows the [commit guidelines](https://github.com/JaKooLit/Hyprland-Dots/blob/main/CONTRIBUTING.md#git-commit-messages).
- [ X] My change requires a change to the documentation.
- [ ] I want to add something in Hyprland-Dots wiki.
- [ ] I have added tests to cover my changes.
- [ X] I have tested my code locally and it works as expected.
- [ X] All new and existing tests passed.

## Screenshots

(if appropriate)

## Additional context
image

### CODE REVIEW

**Comment by JaKooLit** [comment]
thanks for this... Kindly make a PR on development branch.. Ill take a look next week since I will be travelling this week.

thanks

**Comment by Sipper1236** [comment]
> thanks for this... Kindly make a PR on development branch.. Ill take a look next week since I will be travelling this week.
> 
> thanks

ok

File: config/hypr/scripts/Refresh.sh [hunk 1]
```diff
@@ -15,7 +15,7 @@ file_exists() {
 }
 
 # Kill already running processes
-_ps=(waybar rofi swaync)
+_ps=(waybar rofi swaync cava)
```
Comment (JohnRTitor) [inline config/hypr/scripts/Refresh.sh:18]: Here you are killing 1cava`, shouldn't you also restart it below?

File: config/hypr/scripts/GameMode.sh [hunk 2]
```diff
@@ -16,15 +16,22 @@ if [ "$HYPRGAMEMODE" = 1 ] ; then
...
         keyword general:border_size 1;\
         keyword decoration:rounding 0"
-    swww kill 
+    swww kill
+    
+    _ps=(waybar rofi swaync cava)
+for _prs in "${_ps[@]}"; do
+    if pidof "${_prs}" >/dev/null; then
+        pkill "${_prs}"
+    fi
```
Comment (JohnRTitor) [inline config/hypr/scripts/GameMode.sh:25]: - Proper indentation, please.
- Why would you want GameMode to kill waybar?

File: config/hypr/scripts/Refresh.sh (hunk 1 above)
Comment (Sipper1236) [reply config/hypr/scripts/Refresh.sh:18]: idk why but refresh.sh alreddy starts cava again

**Pull request closed without merging**